src/utils.js

```diff
@@ -616,6 +616,69 @@ export const snakeCaseToCamelCase = (snakeCaseStr) => {
         .join('');
 };
 
+/**
+ * Scrolls to the bottom of a page, or until it times out.
```

Comment:
This would deserve a longer and more detailed comment, e.g. see https://github.com/apifytech/actor-legacy-phantomjs-crawler/blob/master/src/phantom_scripts/modules/crawlerutils.js#L767